packages/docs/src/entry.ssr.tsx

```diff
@@ -1,10 +1,11 @@
-import { renderToStream, type RenderToStreamOptions } from '@builder.io/qwik/server';
+import type { PreloaderOptions, RenderToStreamOptions } from '@builder.io/qwik/server';
+import { renderToStream } from '@builder.io/qwik/server';
 import Root from './root';
 
 // You can pass these as query parameters, as well as `preloadDebug`
 const preloaderSettings = [
-  'maxPreloads',
-  'minProbability',
+  'ssrPreloads',
+  'ssrPreloadProbability',
```

Comment:
for consistency I would name it `ssrMinPreloadProbability`, or remove the "min" in the the other.